scripts/AlcorAgents.txt

```diff
@@ -0,0 +1,4 @@
+ip-address
+10.213.43.161
```

Comment:
Let us remove specific internal IP addresses. Instead, we could give simple instruction on how to set this file.